tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
```

Comment:
I'm not familiar with `tox`, but line 32 suggests tox uses 2.7. Unsure if this is an issue?